pkg/cmd/eventlistener/logs.go

```diff
@@ -56,7 +57,12 @@ Show 2 lines of most recent logs from all EventListener pods:
 				return fmt.Errorf("tail cannot be 0 or less than 0 unless -1 for all pods")
 			}
 
-			err := getEventListener(args[0], p)
+			cs, err := p.Clients()
+			if err != nil {
+				return fmt.Errorf("failed to create tekton client")
+			}
+
+			_, err = eventlistener.Get(cs, args[0], metav1.GetOptions{}, p.Namespace())
 			if err != nil {
 				return err
```

Comment:
can you return here `fmt.Errorf("failed to get EventListener %s: %v", elName, err)`

Reply:
instead of returning this error message from here I have returned from Get function in [pkg/eventlistener/eventlistener.go](https://github.com/tektoncd/cli/pull/1455/files#diff-0c51a3f2f3739c3b3f2a0eb7abf7022bf18cac04cd074469d69431bfba401310R65)